tcp-keepalive/README.md

```diff
@@ -22,6 +22,8 @@ Flags:
   -h, --help                help for server
   -i, --interval duration   Interval to send a keepalive message (default 5s)
   -l, --listen string       Listen address and port (default ":8000")
+      --not-send-msg        Server doesn't send keepalive message
```

Comment:
```suggestion
      --silent        Server doesn't send keepalive message
```
How about `--silent` instead of `--not-send-msg`?

Reply:
good